openbb_terminal/stocks/options/op_helpers.py

```diff
@@ -594,3 +594,20 @@ def Vomma(self, change):
             * norm.pdf(self.d1)
             / self._sigma
         )
+
+
+@log_start_end(log=logger)
+def get_dte(chain: pd.DataFrame) -> pd.DataFrame:
+    """
+    Returns a new column containing the DTE as an integer, including 0.
+    Requires the chain to have the column labeled as, expiration.
+    """
+    if "expiration" not in chain.columns:
+        raise ValueError("No column labeled 'expiration' was found.")
+
+    now = datetime.now()
+    temp = pd.DatetimeIndex(chain.expiration)
+    temp_ = (temp - now).days + 1
```

Comment:
If expirarion = now, then this would return 1 instead of 0.

I would like to see this done how I do elsewhere, where you look at the total_seconds between now the expiration (need to add 16 hours) and then convert seconds to days.  This will also avoid the divergence for 0dte calculates.